actions/province_age_form.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict, Text, Any, List
+
+from rasa_sdk import Tracker
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk.forms import FormAction
+
+
+class ProvinceAgeForm(FormAction):
+    def name(self) -> Text:
+
+        return "province_age_form"
+
+    @staticmethod
+    def required_slots(tracker: Tracker) -> List[Text]:
+        """A list of required slots that the form has to fill"""
+
+        return ["province", "elevated_covid_risk"]
```

Comment:
With the new dialogue flows, I'm not sure if this is the right approach to keep track of the risk.